database.rules.bolt

```diff
@@ -86,4 +113,11 @@ path /years/{year} {
 
         path /members/{memberId} is ProjectMember {}
     }
+
+    path /votes/{voteId} is Vote {
+        write() { isAllowedUser() }
```

Comment:
so if you wanted to validate this against user you'd have to do an annoying lookup on current user vs "find data based on field here in other tree"

Reply:
Yeah, I'm gonna do as a follow up